projects/plugins/boost/app/assets/src/js/features/critical-css/critical-css-meta/critical-css-meta.tsx

```diff
@@ -47,31 +42,27 @@ const CriticalCssMeta: React.FC< CriticalCssMetaProps > = ( {
 						'jetpack-boost'
 					) }
 				</div>
-				<ProgressBar progress={ criticalCssProgress } />
+				<ProgressBar progress={ progress || 0 } />
```

Comment:
If I read this correctly then we're explicitly setting progress to `undefined` in `useLocalCriticalCssGenerator`, but now casting it back to `0`? Why not just 0 without casting it back and forth?

Reply:
Fair. f2e15a4b24